perun-core/src/main/java/cz/metacentrum/perun/core/api/ResourcesManager.java

```diff
@@ -816,6 +816,21 @@
 	 */
 	List<Resource> getResourcesWhereUserIsAdmin(PerunSession sess, Facility facility, Vo vo, User authorizedUser) throws InternalErrorException, PrivilegeException, UserNotExistsException, FacilityNotExistsException, VoNotExistsException;
 
+	/**
+	 * Return all resources for the vo where user is authorized as resource manager.
```

Comment:
You should add information to javadoc, that user can be admin directly or indirectly by authorized group. Same for all javadocs in other layers.